app/browser/api/ledgerNotifications.js

```diff
@@ -116,9 +125,10 @@ const onResponse = (message, buttonIndex, activeWindow) => {
       if (buttonIndex === 0) {
         appActions.changeSetting(settings.PAYMENTS_NOTIFICATIONS, false)
       } else if (buttonIndex === 2 && activeWindow) {
+        const addFundsOverlay = shouldShowAddFundsModal(appStore.getState()) ? '?addFundsOverlayVisible' : ''
```

Comment:
Let's reroute `onResponse` through appAction so that we will get state from the reducer.